google-cloud-storage/clirr-ignored-differences.xml

```diff
@@ -21,4 +21,29 @@
         <method>com.google.cloud.storage.BucketInfo$Builder deleteLifecycleRules()</method>
         <differenceType>7013</differenceType>
     </difference>
+    <difference>
+        <className>com/google/cloud/storage/Storage</className>
```

Comment:
This requires a major version bump.

Reply:
> This requires a major version bump.

@frankyn what do you think on this??

Reply:
We've done it for others and updated the README.md to reflect this quirk in updates of the Interface. We haven't hit customer issues with these interface changes.

We are going to move forward with similar changes.